site/src/examples/border-layout/StickyPositioning.tsx

```diff
@@ -0,0 +1,36 @@
+import { ReactElement } from "react";
+import clsx from "clsx";
+import { BorderLayout, BorderItem } from "@salt-ds/core";
+import styles from "./index.module.css";
+import stickyPositioningStyles from "./StickyPositioning.module.css";
+
+export const StickyPositioning = (): ReactElement => (
+  <BorderLayout className={stickyPositioningStyles.borderLayout}>
+    <BorderItem position="north" sticky className={styles.borderItem}>
+      North
+    </BorderItem>
+    <BorderItem position="west" className={styles.borderItem}>
+      West
+    </BorderItem>
+    <BorderItem
+      position="center"
+      className={clsx(styles.borderItem, stickyPositioningStyles.center)}
+    >
+      <p>Center</p>
+      <p>
+        Center Dolor ea veniam velit esse ex nulla non anim officia commodo.
```

Comment:
Not a huge thing but did we ever figure out what we should use instead of lorem ipsum?

Reply:
I'm not sure if we ever agreed on this @bhoppers2008 @mikearildbrown ?